.github/workflows/ci.yml

```
@@ -0,0 +1,43 @@
+name: Java CI with Maven + SonarCloud
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)